### PR TITLE
Rework values for table and table field

### PR DESCRIPTION
**Pull request by Coquinho**
Table start with all CRUD permissions and show it on it's value
Field value now is `<name>\n<cType>` and start with it's default value as
  `field\nuint8_t`
CRUD actions are no more for CPI and DP

### CODE REVIEW

**Comment by Coquinho** [comment]
This uses #13

File: src/main/webapp/js/grapheditor/Shapes.js [hunk 1]
```diff
@@ -3630,18 +3630,18 @@ function getShape(cell) {
...
-    );
-    permission += " Dpath: " + makePermissionString(
+    ) || "None");
+    permission += " Dpath: " + (makePermissionString(
       style.dpCreate,
       style.dpRead,
       style.dpUpdate,
       style.dpDelete,
-    );
+    ) || "None");
```
Comment (JPTIZ) [inline src/main/webapp/js/grapheditor/Shapes.js:3644]: Nice

Btw why not on `makePermissionString`, maybe?

File: src/main/webapp/js/grapheditor/Shapes.js [hunk 2]
```diff
@@ -3658,10 +3658,9 @@ function getShape(cell) {
     } else if (style.shape === 'program') {
       graph.setCellStyles('name', value, [cell]);
       cell.setValue("Program: " + value);
-
-    } else {
-      cell.setValue(value);
+    } else if (style.shape === 'controlPlaneTableCell') {
       graph.setCellStyles('name', value, [cell]);
+      cell.setValue(value + "\n" +style.cType);
```
Comment (JPTIZ) [inline src/main/webapp/js/grapheditor/Shapes.js:3663]:
```suggestion
      cell.setValue(`${value}\n${style.cType}`);
```

Just to make sure JS won't troll us with its messy typesystem.

File: src/main/webapp/js/grapheditor/Shapes.js [hunk 3]
```diff
@@ -3675,8 +3674,8 @@ function getShape(cell) {
         cell.children[0].setValue(style.name + tableValue(style));
       } else if (style.shape === 'program') {
         cell.setValue("Program: " + style.name);
-      } else {
-        cell.setValue(style.name);
+      } else if (style.shape === 'controlPlaneTableCell') {
+        cell.setValue(style.name + "\n" + style.cType);
```
Comment (JPTIZ) [inline src/main/webapp/js/grapheditor/Shapes.js:3678]:
```suggestion
        cell.setValue(`${style.name}\n${style.cType}`);
```

**Review by JPTIZ — COMMENTED** [review]
About changing to only our shapes to sync value and `Name` property: let's find a different approach for that and leave this PR's with the changes you proposed.

I'm betting on something more..."from outside", e.g.: make property have a "syncWithValue", or just having a list of shapes that will sync name <-> value.

**Comment by Coquinho** [comment]
> About changing to only our shapes to sync value and `Name` property: let's find a different approach for that and leave this PR's with the changes you proposed.
> 
> I'm betting on something more..."from outside", e.g.: make property have a "syncWithValue", or just having a list of shapes that will sync name <-> value.

The ideia of a property `SyncWithValue` seems good to me

**Pull request merged**